lib/web_ui/dev/integration_tests_manager.dart

```diff
@@ -361,7 +462,8 @@ class ChromeIntegrationArguments extends IntegrationArguments {
 
 /// Arguments to give `flutter drive` to run the integration tests on Firefox.
 class FirefoxIntegrationArguments extends IntegrationArguments {
-  List<String> getTestArguments(String testName, String mode) {
+  List<String> getTestArguments(String testName, String mode,
+      {bool isCanvaskitBackend = false}) {
```

Comment:
Does `isCanvaskitBackend` need a default value? Do we ever not pass it? If we always pass it, consider removing the default and making it `@required` (or `required` if this is null-safe code)

(this comment applies throughout the PR)

Reply:
removed the named parameter all together, looks like it wasn't adding extra benefit.